src/test/unit_tests/jest_tests/test_nc_nsfs_account_cli.test.js

```diff
@@ -525,7 +525,7 @@ describe('manage nsfs cli account flow', () => {
             };
 
             beforeEach(async () => {
-                await P.all(_.map([CONFIG_SUBDIRS.ACCOUNTS, CONFIG_SUBDIRS.ACCESS_KEYS], async dir =>
+                await P.all(_.map([ CONFIG_SUBDIRS.ROOT_ACCOUNTS, CONFIG_SUBDIRS.ACCESS_KEYS], async dir =>
```

Comment:
I think this space is redundant
```suggestion
                await P.all(_.map([CONFIG_SUBDIRS.ROOT_ACCOUNTS, CONFIG_SUBDIRS.ACCESS_KEYS], async dir =>
```